### ISSUE TITLE
SteamOS Screen reader

### ISSUE THREAD

**Issue by Ghorthalon** [ValveSoftware/SteamOS#833]
#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen.

I would like screen reader accessibility in SteamOS/Steam Big Picture.
I believe that the controller-based interface would lend itself very well to being usable without sight.
As a blind player I would like to be able to make full use of my SteamDeck, including SteamOS. Currently, I can only play games on my SteamDeck if I install a third-party operating system which includes screen reading functionality, such as Windows. Even then, certain features, like the controller configuration, remain completely inaccessible even on Windows, so I cannot change the configuration of the controller such as trackpads.
Linux provides a system for text-to-speech called Speech Dispatcher, which could handle TTS for the SteamOS UI.

#### Steps for reproducing this issue:

1. Boot up any SteamOS, or try to initiate the controller configuration in Steam under Windows in library -> some game -> manage -> controller configuration.
2. No speech support, meaning without sight, I cannot change anything about the controller configuration, or use the SteamOS UI.

**Comment by syberphunk** [comment]
I agree with this. It would also be beneficial to have a way to support MS Sappi (Microsoft Speech API / Microsoft Narrator) for games which support it also. Here's an example list of games that support MS Sappi:

Games such as:
Eagle Island
Blind Drive
Alt Frequencies
1428:Shadows Over Silesia
Code 07
Kilta
Bad Faith

There are also potentially linux screen readers such as Orca, Speech Dispatcher,  and Espeak. These need to be considered alongside.

**Comment by eric-blue** [comment]
I'm surprised this hasn't got more support. I am thankful to have a macbook with stellar screen reader support but would struggle if my steam deck was my only PC, not strictly for gaming. 

For instance, I use screen readers, not for sight but because I'm a very slow reader, so having text-to-speech is a life changer. Valve should see that there is a substantial userbase between our use cases that they are ignoring

**Comment by syberphunk** [comment]
Yes, I would also love for this to get more attention.

**Comment by My1** [comment]
or even just a way to have a third party screenreader like orca possible, could that even run in a flatpak environment?

**Comment by LeifAndersen** [comment]
I've been looking into this, and it actually looks pretty easy to do (relatively speaking). The SteamOS gui seems to be more or less CEF (chromium embedded framework), and looking at the DOM it creates it even has some of the `aria` tags already.

I'm going to try making a plugin for it to enable a screen reader. Although it would be great if Valve was interested in working with me to get it into the core accessibility menu.

**Comment by rings2006** [comment]
hows this going? really a steam client accessibility fix would be nice in general, its literally the one thing keeping me on windows

**Comment by kisak-valve** [comment]
Closing as implemented.

**Comment by syberphunk** [comment]
@kisak-valve You're awesome.
